src/Conda.jl

```diff
@@ -34,43 +34,95 @@ module Conda
 using Compat
 import Compat.String
 using JSON
+include("../deps/deps.jl")
 
-"Prefix for installation of all the packages."
-const PREFIX = abspath(dirname(@__FILE__), "..", "deps", "usr")
+if !isdir(default_dir)
+    # Ensure default_dir exists
+    mkpath(default_dir)
+end
+
+immutable Environment
```

Comment:
I'm not sure we want to declare a type that is nothing more than a wrapper around a pathname.